src/actions/index.js

```diff
@@ -40,20 +41,29 @@ export function addCard(newCard) {
       .then(body => {
         return dispatch({
             type : ADD_CARD,
-            payload : newCard
+            payload : body
           });
       });
   }
 }
 
-export function deleteCard(card) {
+export function deleteCard(card_id) {
   return (dispatch) => {
-    return fetch('/', {
+    return fetch('/cards', {
       method : 'DELETE',
-      body : JSON.stringify(card),
+      body : JSON.stringify(card_id),
```

Comment:
You don't need to send a body for delete, just put the card_id in the path for fetch.